tests/functional/disaster-recovery/regional-dr/conftest.py

```diff
@@ -20,3 +25,14 @@ def pytest_collection_modifyitems(items):
                     f"Test {item} is removed from the collected items. Test runs only on RDR clusters"
                 )
                 items.remove(item)
+
+
+@pytest.fixture(autouse=True)
+def check_subctl_cli():
+    # Check whether subctl cli is present
```

Comment:
Should we run this only for RDR scenario? If so, then we should add condition and return at the start of the fixture if it's not RDR setup I guess and do not enforce subctl for non RDR clusters where we don't need it.

Reply:
I agree with Petr's suggestion